pipelines.tf

```diff
@@ -141,6 +141,27 @@ module "github_webhook_report" {
   admin_role                  = local.tf_admin_role
 }
 
+module "monitor_aws_iam" {
+  source = "./modules/lambda_docker_pipeline"
+
+  pipeline_name = "monitor-aws-iam"
+  src_repo      = "monitor-aws-iam"
+  branch        = "BAU-fix-build"
```

Comment:
`BAU-fix-build` branch? Is this intentional?

Reply:
yes I had needed to make changes to the lambda as errors arise from a different environment. will change this to main in an upcoming PR